tests/certification/state/mysql/README.md

```diff
@@ -0,0 +1,32 @@
+# MySQL State Store certification testing
+
+This project aims to test the MySQL State Store component under various conditions. Tests are run against both MySQL 8 and MariaDB 10.
+
+## Test plan
+
+## Test for CRUD operations
+
+1. Able to create and test connection.
+2. Able to do set, get, update and delete.
+3. Negative test to fetch record with key, that is not present.
+
+## SQL Injection
+
+* Not prone to SQL injection on write
+* Not prone to SQL injection on read
+* Not prone to SQL injection on delete
+
+## Connection Recovery
+
+1- When MySQL/MariaDB goes down and then comes back up - client is able to connect
+
+## Concurrency
```

Comment:
These cases should already be covered by conformance tests.

Reply:
Ok, to leave here - but please be mindful of cert test bloat. Conformance tests already cover a lot. This makes maintenance harder.

Reply:
Gotcha. I based the test on the Postgres one, I can remove them if needed